source/Conversation.cpp

```diff
@@ -361,7 +369,17 @@ Conversation Conversation::Instantiate(map<string, string> &subs, int jumps, int
 	for(Node &node : result.nodes)
 	{
 		for(Element &element : node.elements)
+		{
+			if(element.isPhrase)
+			{
+				const Phrase * phrase = GameData::Phrases().Find(element.text);
+				if(phrase)
+					element.text = phrase->Get();
+				element.text += "\n";
+				element.isPhrase = false;
```

Comment:
why?

Reply:
Why what?

Reply:
if we keep this syntax we should rename isPhrase to holdsPhrase or smth like that; its still a phrase that's contained and idk why we would mark it as not a phrase all of a sudden